### PR TITLE
AWS Marketplace for Dedicated

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2211
Review deadline: Monday, July 22

## Page previews
[What's New in Redpanda Cloud](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/)
[Sign up for Dedicated](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#dedicated-cloud)
[Serverless vs Dedicated/BYOC](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#serverless-vs-dedicatedbyoc)
[Serverless overview](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/cloud-overview/#serverless) (adds billing activity)
[Serverless: Unsupported features](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/serverless/#unsupported-features)

## Checks

- [ ] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 1]
```diff
@@ -34,7 +34,9 @@ When you create a Dedicated cluster, you select the supported xref:deploy:deploy
 
 ==== Sign up for Dedicated
 
-To start using Dedicated, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer for monthly or annual committed use. With a usage-based billing commitment, you sign up for a monthly or an annual minimum spend amount. You can then use xref:deploy:deployment-option/cloud/manage-billing/gcp-commit.adoc[Google Cloud Marketplace] or xref:deploy:deployment-option/cloud/manage-billing/aws-commit.adoc[AWS Marketplace] to quickly provision Dedicated Cloud clusters, and you can view invoices and manage your subscription in the marketplace.
+To start using Dedicated, sign up for a free trial on the AWS Marketplace. New trials receive $300 (USD) in free credits to spend in the first 30 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using Dedicated, add a credit card and pay as you go.
```
Comment (vladoschreiner) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: AFAIK credit card isn't applicable for marketplace trials. Marketplace sign-ups pay via their cloud account (we charge them via marketplace). Not sure whether we mandate the payment method they use in the marketplace.

Also, no action is needed to extend the trial. Once they burn free credits or the free credits expire, they are just charged via marketplace. They are charged unless they cancel the subscription. 

@towfiqa to answer.

**Review by vladoschreiner — CHANGES_REQUESTED** [review]

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc (hunk 1 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]: Thanks, @vladoschreiner! I've updated this and added to the [What's New for Cloud](https://deploy-preview-612--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/whats-new-cloud/).

**Review by towfiqa — APPROVED** [review]
Reviewed everything related to Dedicated bits and they look good!

File: modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc [hunk 2]
```diff
@@ -17,13 +17,13 @@ NOTE: With standard BYOC clusters, Redpanda manages security policies and resour
...
 
 Available on AWS.
 
 ==== Sign up for Serverless
 
-To start using Serverless, https://redpanda.com/try-redpanda/cloud-trial#serverless[sign up for a free trial^]. New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using Serverless, add a credit card and pay as you go. You can edit payment methods on the Redpanda Cloud *Billing* page. 
+To start using Serverless, https://redpanda.com/try-redpanda/cloud-trial#serverless[sign up for a free trial^]. New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using your Serverless cluster, add a credit card and pay as you go.
```
Comment (Feediver1) [inline modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc]:
```suggestion
To start using Serverless, https://redpanda.com/try-redpanda/cloud-trial#serverless[sign up for a free trial^]. New trials receive $100 (USD) in free credits to spend in the first 14 days. This should be enough to run Redpanda with reasonable throughput. No credit card is required for a trial. To continue using your Serverless cluster after the free trial, add a credit card and pay as you go. 
```

**Review by Feediver1 — APPROVED** [review]

**Comment by vladoschreiner** [comment]
Please hold on. I just got informed by eng that the billing UI feature isn't ready: https://github.com/redpanda-data/cloudv2/issues/15999

@frankieshakes to confirm feature readiness 

LGTM otherwise

**Comment by frankieshakes** [comment]
Phase 1 work is complete. A bug fix for [issue #17087](https://github.com/redpanda-data/cloudv2/issues/17087) was just pushed out to integration. A release to preprod and production should be cut later this evening.

Phase 2 work is set to start next week (week of Aug 5th).

**Review by vladoschreiner — APPROVED** [review]
LGTM

**Pull request merged**